yt_dlp/extractor/globo.py

```diff
@@ -38,126 +33,70 @@ class GloboIE(InfoExtractor):
             'ext': 'mp4',
             'title': 'Acidentes de trânsito estão entre as maiores causas de queda de energia em SP',
             'duration': 137.973,
-            'uploader': 'Rede Globo',
-            'uploader_id': '196',
+            'uploader': 'Bom Dia Brasil',
+            'uploader_id': '810',
         },
         'params': {
             'skip_download': True,
         },
-    }, {
-        'url': 'http://canalbrasil.globo.com/programas/sangue-latino/videos/3928201.html',
-        'only_matching': True,
-    }, {
-        'url': 'http://globosatplay.globo.com/globonews/v/4472924/',
-        'only_matching': True,
-    }, {
-        'url': 'http://globotv.globo.com/t/programa/v/clipe-sexo-e-as-negas-adeus/3836166/',
-        'only_matching': True,
-    }, {
-        'url': 'http://globotv.globo.com/canal-brasil/sangue-latino/t/todos-os-videos/v/ator-e-diretor-argentino-ricado-darin-fala-sobre-utopias-e-suas-perdas/3928201/',
-        'only_matching': True,
-    }, {
-        'url': 'http://canaloff.globo.com/programas/desejar-profundo/videos/4518560.html',
-        'only_matching': True,
```

Comment:
If we are removing all these tests, does that mean these URL formats are no longer valid? If so, then we should tighten up the `_VALID_URL` regex too